.github/workflows/try-runtime-tests.yml

```diff
@@ -0,0 +1,30 @@
+name: AHM try-runtime tests
```

Comment:
@kianenigma I add this entrypoint to dispatch and use a common like we have in other workflows.
This was the missing workflow, that allow to pass the run-id or download the latest one automatically.
Thx!